scrapy/utils/reactor.py

```diff
@@ -49,13 +49,21 @@ def __call__(self):
         return self._func(*self._a, **self._kw)
 
 
-def install_reactor(reactor_path):
+def install_reactor(reactor_path, event_loop_path):
     """Installs the :mod:`~twisted.internet.reactor` with the specified
-    import path."""
+    import path.Also installs asyncio event loop as specified in the import
+    path if asyncio reactor is enabled"""
     reactor_class = load_object(reactor_path)
     if reactor_class is asyncioreactor.AsyncioSelectorReactor:
         with suppress(error.ReactorAlreadyInstalledError):
-            asyncioreactor.install(asyncio.get_event_loop())
+            if event_loop_path is not None:
+                x = __import__(event_loop_path)
```

Comment:
I think you should use `import_module`. And I think you should not check if it is `None`, you should use whatever is loaded, and let it fail if the value is wrong.

And maybe `x` should be renamed to `event_loop_module` or something in that line. And `loop` to `event_loop` for consistency.